apps/website/.vuepress/theme/components/Sidebar.vue

```diff
@@ -126,6 +123,10 @@
 .nav-group-children {
   overflow-y: hidden;
   transition: height 0.2s ease-in-out;
+
+  &.is-expanded {
+    height: fit-content !important;
```

Comment:
If this is fixing the issue, we should probably add a comment here why `!important` is used here.

Reply:
I added a comment. It's not clear why and I can't spend more time debugging it at the moment. It should let me finish the release and publish the website.